wren-ui/Dockerfile

```diff
@@ -58,4 +61,4 @@ ENV PORT 3000
 
 # server.js is created by next build from the standalone output
```

Comment:
https://yarnpkg.com/migration/guide#use-yarn-dlx-instead-of-yarn-global

should we use `yarn dlx` on line 56? (There is no diff so I comment on this line)

Reply:
I ran into this error while building an image on my Mac (m1).
<img width="1182" alt="image" src="https://github.com/user-attachments/assets/c57e5017-7276-40d1-96a9-b9fbc408a3ce" />